internal/metamorph/processor.go

```diff
@@ -316,6 +318,9 @@ func (p *Processor) updateMined(ctx context.Context, txsBlocks []*blocktx_api.Tr
 		if len(data.Callbacks) > 0 {
 			p.callbackSender.SendCallback(ctx, data)
 		}
+
+		// remove tx from cache - we don't expect any more status updates
+		p.delTxFromCache(data.Hash)
```

Comment:
I think we can already delete the tx from cache when it was seen on network, don't you think?

Reply:
You're right, removed here.